app/javascript/components/ArtistWorks.js

```diff
@@ -48,37 +50,82 @@ class ArtistWorks extends React.Component {
     Requester.post(commissions_route, payload)
   }
 
-  createNewWork() {
+  createNewWork = () => {
     window.location = `/works/new`;
   }
 
+  renderFilterButtons = () => {
+    const { activeFilter } = this.state;
+    return (
+      <div className="mt5 mb3">
+        {['All works', 'Available', 'Sold/Rented'].map(filter => {
+          const className = activeFilter == filter
+            ? "filter-button bg-gray white f6 mr3"
+            : "filter-button bg-white gray f6 mr3";
+          return <button onClick={() => this.setState({ activeFilter: filter })} key={filter} className={className}>{filter}</button>
+        })}
+      </div>
+    )
+  }
+
   render() {
+    const { name, program } = this.state.artist;
     return (
       <div>
-        <button onClick={this.createNewWork}>
-          New Work
-        </button>
-        <h2>These will be the artist works</h2>
-        {this.state.works.map(work => (
-          <div key={work.id}>
-            <h3>{work.title}</h3>
-            <p>{work.work_type}</p>
-            <p>{work.media}</p>
-            {work.attachment_url.map((attachment) =>
-              <img src={attachment} width="200" height="200"/>
-            )}
+        <h1> {name} </h1>
+        <div className="row-bio flex">
+          <div className="pa3 flex flex-column w5 bg-white">
+            <div className="h4 w4 br-100 mb3 bg-gray self-center">
+              <img src="" />
+            </div>
+            <div className="info">
+              <h3> Education </h3>
+              <p> {program} </p>
+            </div>
+            <div className="mt-auto self-center">
+              <button className="contact-button f5">contact</button>
+            </div>
+          </div>
+          <div className="bg-gray mh3 flex-auto">
```

Comment:
Just one last comment--could we assign a min-width to this? When the window is readjusted, everything else will stay on the window except this component (which horizontally shrinks into nothing).